<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DesignDialog</class>
 <widget class="QDialog" name="DesignDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>434</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox0">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>90</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>0.700000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox1">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>0.400000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>90</y>
     <width>70</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter the desired concentration of the three pipes (from left to right, decreasing successively)</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DesignDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DesignDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
